src/pages/home/report/ReportActionsList.js

```diff
@@ -78,10 +78,10 @@ function keyExtractor(item) {
 const ReportActionsList = (props) => {
     const opacity = useSharedValue(0);
     const animatedStyles = useAnimatedStyle(() => ({
```

Comment:
We should use our `OpacityView` utility component here instead of using Reanimated directly.

Reply:
The OpacityView does the opposite actually - it dimms the view, here the appears. Shoul I extend the OpacityView component then?

Reply:
nah, this is fine